now-playing/frontend/command-manager/command-np/engine/__init__.py

```diff
@@ -0,0 +1,26 @@
+#-*- coding: utf-8 -*-
+#
+# DO WHAT YOU WANT TO PUBLIC LICENSE
+#
+# Copyright (c) 2015 James Axl <axlrose112@gmail.com>
+#
+# Everyone is permitted to copy and distribute verbatim or modified
+# copies of this license document, and changing it is allowed as long
+# as the name is changed.
+#
+# DO WHAT YOU WANT TO PUBLIC LICENSE
+# TERMS AND CONDITIONS FOR COPYING, DISTRIBUTION AND MODIFICATION
+#
+# 0. You just DO WHAT YOU WANT TO.
+# 1. Earth and all that is in it, with knowledge we protect it. And the best of mankind knowledge makes him and he makes it.
```

Comment:
Don't change the license terms please, simply copy/paste it from the original now-playing hook